packages/pelagos/src/listInput/ListEntries.js

```diff
@@ -16,14 +19,24 @@ const ListEntries = ({
 	className,
 	highlightKey,
 	list,
+	reorderable,
 	column,
 	getItemKey,
 	getItemName,
 	renderItem,
+	onReorder = moveListItem,
 	onRemoveClick,
 	onHighlightClear,
 }) => {
-	const liveRef = useRef(null);
+	const getElementName = useCallback((element) => getItemName(list[+element.dataset.index]), [list, getItemName]);
+	const updateList = useCallback((fromIndex, toIndex) => onReorder(list, fromIndex, toIndex), [list, onReorder]);
```

Comment:
You have to call `onReorder` with the *result* of the call to `moveListItem` so the caller can store the new list.